src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Resources/Theme/Dark.xaml

```diff
@@ -440,6 +440,51 @@
 
     <!--  DataGrid  -->
+    <SolidColorBrush x:Key="DataGridColumnFloatingHeaderBorderBrush" Color="{StaticResource ControlStrongStrokeColorDefault}" />
+    <SolidColorBrush x:Key="DataGridHeaderDropSeparatorBackground" Color="{StaticResource ControlStrongStrokeColorDefault}" />
+    <SolidColorBrush x:Key="DatatGridSelectAllButtonForeground" Color="{StaticResource TextFillColorPrimary}" />
+    <SolidColorBrush x:Key="DataGridCellFocusVisualBorderBrush" Color="{StaticResource TextFillColorSecondary}" />
+    <SolidColorBrush x:Key="DataGridPillFillBrush" Color="{StaticResource SystemAccentColorDark1}" />
+    <SolidColorBrush x:Key="DataGridRowValidationErrorForeground" Color="{StaticResource SystemFillColorCritical}" />
+    <SolidColorBrush x:Key="DataGridBackground" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridBorderBrush" Color="{StaticResource ControlStrokeColorDefault}" />
+    <SolidColorBrush x:Key="DataGridForeground" Color="{StaticResource TextFillColorPrimary}" />
+    <SolidColorBrush x:Key="DataGridLinesBrush" Color="{StaticResource ControlStrokeColorDefault}" />
+    <SolidColorBrush x:Key="DataGridHeaderSeparatorBrush" Color="{StaticResource ControlStrokeColorSecondary}" />
+    <SolidColorBrush x:Key="DataGridRowBackground" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridRowBorderBrush" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridRowBackgroundPointerOver" Color="{StaticResource SubtleFillColorTertiary}" />
+    <SolidColorBrush x:Key="DataGridRowBorderBrushPointerOver" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridRowBackgroundSelected" Color="{StaticResource SubtleFillColorSecondary}" />
+    <SolidColorBrush x:Key="DataGridRowBorderBrushSelected" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridCellBackground" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridCellBorderBrush" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridCellBackgroundPointerOver" Color="{StaticResource SubtleFillColorSecondary}" />
+    <SolidColorBrush x:Key="DataGridCellBorderBrushPointerOver" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridCellBackgroundSelected" Color="{StaticResource SubtleFillColorTertiary}" />
+    <SolidColorBrush x:Key="DataGridCellBorderBrushSelected" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridCellBackgroundInactiveSelection" Color="{StaticResource SubtleFillColorTertiary}" />
+    <SolidColorBrush x:Key="DataGridCellBorderBrushInactiveSelection" Color="{StaticResource SubtleFillColorTransparent}" />
+    <SolidColorBrush x:Key="DataGridCellBackgroundFocused" Color="{StaticResource SubtleFillColorTransparent}" />
```

Comment:
In dark mode, should this textbox have a white background?

![image](https://github.com/user-attachments/assets/51d23c90-6d9b-482a-84f5-4abd62b33e21)